specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storageTaskAssignments.json

```
@@ -0,0 +1,840 @@
+        "tags": [
+          "StorageTaskAssignments"
+        ],
+        "description": "Fetch the report summary of a single storage task assignment's instances",
+        "operationId": "StorageTaskAssignmentInstancesReport_List",
+        "x-ms-examples": {
+          "ListStorageTaskAssignmentInstancesReportSummary": {
+            "$ref": "./examples/storageTaskAssignmentsList/ListStorageTaskAssignmentInstancesReportSummary.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "#/parameters/StorageTaskAssignmentName"
+          },
+          {
+            "name": "$maxpagesize",
+            "in": "query",
+            "type": "string",
+            "description": "Optional, specifies the maximum number of storage task assignment instances to be included in the list response."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "type": "string",
+            "description": "Optional. When specified, it can be used to query using reporting properties."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- storage task assignment's instances report summary retrieved and returned successfully",
+            "schema": {
+              "$ref": "./common.json#/definitions/StorageTaskReportSummary"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StorageTaskAssignment": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StorageTaskAssignmentProperties",
+          "description": "Properties of the storage task assignment."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The storage task assignment."
+    },
+    "StorageTaskAssignmentProperties": {
+      "type": "object",
+      "properties": {
+        "taskId": {
+          "type": "string",
```

```suggestion
          "type": "string",
          "format": "arm-id"
```
To clarify it is a fully qualified resource id, perhaps? Noticed we're using this in storage actions swagger